### DISCUSSION TITLE
Open Community Working Meetings

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/community#35] [Announcements]
# Monthly

Open Community Working Meetings are a 1 hour, agenda'd only, with a focus on furthering the JSON Schema specification and organization.

**The calls are at 1:00 PT Monthly every 3rd Monday.**

All meetings are made public and usually result in a number of action items.  

Everyone is welcome to join the meeting, or add new discussion points in the agenda. 
You can have a PR or an issue added to the agenda of the next Working Meeting by just adding the `agenda` label.

All calls are subject to our [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).

# Calendar and Joining links

Just want to join the call? Here's the google meet link : https://meet.google.com/raz-ptdn-ewf.

You can find these events scheduled on our **[JSON Schema Community Calendar](https://calendar.google.com/calendar/embed?src=info%40json-schema.org&ctz=Europe/London)**. 

To add the events to your Google Calendar, use the link found in the bottom right of the above calendar.
To add the events to another calendar, use the provided [ics/ical file](https://calendar.google.com/calendar/ical/info%40json-schema.org/public/basic.ics).

We will aim for these calls to have an agenda before the call starts, but we want to be flexible as we feel our way into these regular calls.

These calls will be recorded and made publically accessible.

Further links:
* [Currently scheduled meetings or meetings with unresolved action items](https://github.com/json-schema-org/community/issues?q=is%3Aissue+label%3A%22Working+Meeting%22+is%3Aopen)
* [Resolved meetings](https://github.com/json-schema-org/community/issues?q=is%3Aissue+label%3A%22Working+Meeting%22+is%3Aclosed)
* [Meeting in Youtube](https://www.youtube.com/watch?v=48S8-GwRh-g&list=PLHVhS4Tj1YZPYt6sMkvf4nW8zKvZExVA4)

**Comment by benjamin-mogensen** [comment]
Any of these for 2022?

**Reply by Relequestual** [reply]
Yes @benjamin-mogensen. Use the registration link above.
We have had four calls so far this year.
You can see them here: https://github.com/json-schema-org/community/issues?q=is%3Aissue+label%3A%22Working+Meeting%22+
All but the latest have links to the recording at the bottom.

**Comment by benjagm** [comment]
Hi everyone, we needed to update the Calendar urls because of upgrading our gmail account for our community meetings. New URLs are:
- Calendar: https://calendar.google.com/calendar/embed?src=json.schema.community%40gmail.com&ctz=Europe/London
- iCal: https://calendar.google.com/calendar/ical/json.schema.community%40gmail.com/public/basic.ics

Apologies for the inconvenience.
